src/transaction.ts

```diff
@@ -1954,6 +1955,9 @@ export class Transaction extends Dml {
     optionsOrCallback?: CommitOptions | CallOptions | CommitCallback,
     cb?: CommitCallback
   ): void | Promise<CommitResponse> {
+    const span = getActiveOrNoopSpan();
+    span.addEvent('Starting Commit');
```

Comment:
I know this is part of the scope, but as long as we add this method in "tracewrapper" , this event is not required, same goes for below event `Commit failed` , if the error gets sets, then its not requried.